Libraries/src/Amazon.Lambda.LexEvents/LexEvent.cs

```diff
@@ -155,6 +155,11 @@ public class SlotDetail
             /// The resolutions array contains a list of additional values recognized for the slot.
             /// </summary>
             public IList<Dictionary<string, string>> Resolutions { get; set; }
+
+            /// <summary>
+            /// The originalValue field contains the value that was entered by the user for the slot.
+            /// </summary>
+            public string OriginalValue { get; set; }
```

Comment:
Update the following test to check this value. https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/test/EventsTests.Shared/EventTests.cs#L1063

Looks like the test JSON already has a value set for `original-value` https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/test/EventsTests.Shared/lex-event.json#L21

Reply:
Done